lib/brain.js

```diff
@@ -1368,6 +1378,15 @@ Brain.prototype._billRead = function _billRead (data) {
 
     return
   }
+  // TODO: Check if threshold is 0 and popup the sms screen without bill insertion
+  if(this.trader.smsVerificationActive && fiat.gte(this.trader.smsVerificationThreshold)) {
```

Comment:
Looks like there's no code to reject the bill, which is part of the spec. The bill should be rejected, which will clear ``this.bill``, which should prevent the error you were seeing when subsequently inserting bills.